src/main/scala/ru/org/codingteam/horta/plugins/markov/MarkovUser.scala

```diff
@@ -47,7 +49,7 @@ class MarkovUser(val room: String, val nick: String) extends Actor with ActorLog
       val network = getNetwork()
 
       def generator() = {
-        val phrase = generatePhrase(network, length)
+        val phrase = generatePhrase(network, length)(credential)
```

Comment:
This diff looks strange. You made the `credential` implicit but pass it explicitly. Actually, I'm ok with the changes. Just saying.